### ISSUE TITLE
Project 3: Environmental impact on fish distribution in the Gulf of Mexico over the last three decades

### ISSUE THREAD

**Issue by mbjones** [NCEAS/oss-2017#37]
Author: **Author 1**, **Author 2**, **Author 3**, ...
Topics: list of topics, comma separated

## Summary of Synthesis

How are fish populations distributed in the Gulf of Mexico, and what factors are responsible for this distribution? 

Develop statistical models to relate fish distributions to environmental factors and fishing effort. Environmental factors will include temperature, salinity, dissolved oxygen, chlorophyll a, mixed layer depth, and maps of reef/oil rig locations. 

We will focus on two federally managed species: an invertebrate species (shrimp) and a fish species (red snapper) as examples.

We will develop an open source analytical framework that can be applied to other species in the GOM.

## Research Questions

1. How does the environment affect the distribution of fisheries species in the northern Gulf of Mexico?

- Is there a simple suite of environmental factors that can be used to predict the distribution of fisheries species within the GOM?

2. Where are the major gaps in our regional understanding of the distribution and abundance of GOM species (red snapper and shrimp), specifically in terms of monitoring, restoration, and management efforts?

-  Compare predicted species distributions (from question number 1) with geographic distributions of monitoring, restoration, and management efforts. Where are the areas in the Gulf that lack fisheries data for specific species?

3. How have the environment and fish distribution in the Gulf of Mexico changed over time (from 1982-2016?), and how has this impacted distribution of red snapper and shrimp?

## Rationale

Understanding the relationship between fish distribution and the environment is at the very core of our ability to predict a species’ persistence in an ecosystem. 

Current species distribution/habitat quality maps aren’t ideal because habitat is often defined broadly over large regions but does not take into account physical variables, spatial dynamics, or availability of resources in the form of lower trophic levels (phytoplankton).

R1:
Determining the effect of environmental factors on the distribution of fisheries species facilitates incorporation of fisheries management. Including a spatial component into evaluating environmental effects on fish and fisheries is new and important, too: red tide may affect mortality or spawning success of red grouper if the red tide event overlaps with red grouper spawning grounds, but may not if it does not. Spatial analyses would allow us to map the distribution of important fisheries species, and relate those to the distribution of environmental factors such as bottom dissolved oxygen, salinity, temperature, habitat type, and Chlorophyll a, and to the distribution of fishing effort. The strength of the correlations will provide insight into which factors are important drivers of fish distribution. 

Current spatial physical/biological models (simulating e.g. DO, Chl a) and ecosystem models (simulating e.g. fish biomass, distribution, and landings) only have point calibration/validation. Maps and methods to compare observed vs. predicted could immediately be used for spatial validations of these models. Maps of environmental factors could immediately be used as environmental drivers in ecosystem models, (especially ones for which we have determined are important drivers of fish distribution).

R2:
Improving regional monitoring, restoration, and management efforts for commercially important (and protected?) species in the GOM requires a comprehensive understanding of: (1) species distributions in relation to habitat characteristics and environmental conditions, and; (2) the distribution  of monitoring and fisheries data and efforts. Likewise, a comprehensive understanding of these interactions is crucial for predicting impacts of climate change and anthropogenic stressors (e.g. oil spills) on the distribution/abundance/productivity of these species.

R3:
The environment is changing; especially in response to climate change. Determining which factors are mostly driving fish distribution, how they are changing through time (in the last 30 years), and how fish distribution have changed through time in response to that.

## Potential Deliverables

Minimum viable products:

R1: Fish distribution maps overlaid on the environmental factors that we determined are important drivers of their distribution

R2: Maps of predicted fish distribution based on environmental factors, overlaid with where monitoring effort is concentrated - gives indication whether effort is distributed in a smart way

R3: Fish distribution snaphots over time (e.g. each summer since 1986) overlaid on which environmental factors are mostly responsible for the changes seen over time

Possible future products:

Statistical models for other fisheries species, e.g.:

-	Lionfish
-	Menhaden
-	Groundfish
-	Red drum
-	Grouper

Habitat quality maps- based on relationships between the environment, fishing effort, and fish distribution
	
Species tolerance curves- based on relationships between the environment, fishing effort, and fish distribution
	
Analysis of overlap between water quality (red tide, HAB) and spawning locations.

## Data needs (we have a google docs with sources):
Data Portals
https://modis.gsfc.nasa.gov/data/ MODIS data (Aqua and Terra)
https://podaac.jpl.nasa.gov/ physical oceanography distributed active archive center
http://www.ndbc.noaa.gov/ National Data Buoy Center
http://garmap.gsmfc.org Gulf Artificial Reef Monitoring Program
http://biogeodb.stri.si.edu/caribbean/ Species Distributions of Fishes in the GOM
http://www.iobis.org Ocean Biogeographic Information System (species distributions)
http://www.gbif.org/species Global Biodiversity Information Facility
For example, see http://www.gbif.org/species/2384865,  http://www.gbif.org/species/106008838 for red snapper
https://scihub.copernicus.eu/ Sentinel-1,-2,-3 data portal (SAR satellite, multispectral satellite, etc.)
http://products.gcoos.org/en/   GCOOS data portal with physical, biological, etc

Datasets

https://www.boem.gov/Gulf-of-Mexico-Deepwater-Bathymetry/ - High resolution bathymetry of Northern Gulf of Mexico
 http://seamap.gsmfc.org Gulf States Marine Fisheries Commission (GSMFC) SEAMAP Groundfish (SEAMAP also has point data of DO, Temp, Salinity)
Red snapper CPUE - Cassie Glaspie has data from 94 fish trawls (mix of midwater and bottom water) collected in the NGOMEX for summer 2006-2008, with associated DO, temp, sal, effort.
Survey. Data available from
Seafloor bottom type  - Alex Sacco has from Navy, covers most, if not all of GOM
Seafloor bottom type: http://instaar.colorado.edu/~jenkinsc/dbseabed/
HYCOM data - covers surficial oceanic variables (modeled) http://www.hycom.org
NASA MODIS Chl-a data (low res) https://podaac.jpl.nasa.gov/dataset/MODIS_Aqua_L3_CHLA_Daily_4km_R

Fisheries effort distribution for shrimp (NOAA)

## Analysis

Spatial datasets will be combined in QGIS, and statistical models developed in R.

Validation method: leave one out cross validation

**Comment by demutsertlab** [comment]
I think this one could be a fish and fisheries synthesis project. I should be able to find some time tonight to take a stab at putting some information under the headings

**Comment by Yassin22** [comment]
Is it possible to combine water quality with fishery? Water quality could have a direct impact on fishery. 
Here are the project I thought, could be classified as water quality related projects:
28- A synthesis of multiple datasets describing water quality of coastal regions in the Gulf of Mexico 
25-Algae Blooms in the Gulf of Mexico: monitoring, prediction, impact, and mitigation 
24-A synthesis of research on the role of bacteriophages in oil degradation in wetlands (not sure if this one fits)
23-An analysis of the relevant microbial communities associated with hydrocarbons in the Gulf Region
12-Evaluating the magnitude and interactions of drivers affecting oyster fisheries in the Gulf of Mexico following the Deepwater Horizon oil spill
10-Expansive hypoxia in the Northern Gulf of Mexico

**Fishery Projects:**
33- Climate change, fisheries, and disturbance in the Gulf of Mexico
30-Comprehensive fisheries habitat mapping
19-Fish population abundance and oceanic variation
18-Dolphin-shrimp interaction analysis
13-Assessing vulnerability of fish spawning aggregations to fishing in the Gulf of Mexico 
12-Evaluating the magnitude and interactions of drivers affecting oyster fisheries in the Gulf of Mexico following the Deepwater Horizon oil spill

**Comment by blombergbn** [comment]
I agree there are certainly several fisheries-focused proposals that could comprise a synthesis group. My quick identification includes proposals #10 (to me, while this is about hypoxia, the primary goal is to examine impacts of hypoxia on fisheries...? @demutsertlab ), #13 , #18 , #19 , #30 , #33 , and part of #12 (my proposal, though would be more interested to examine employment vulnerability in relation to social communities rather than diversion of fishing pressure) .

**Comment by blombergbn** [comment]
I think Water Quality, or Environmental Health, could be another synthesis group. For the WQ proposals identified by @Yassin22 , I would add #27 . I'm not sure that #24 fits, but topic could be expanded to Water and Soil Quality - or Environmental Health. For #12 (my proposal), water quality is certainly important for oysters, though I imagined the focus to be more on diversion of fishing pressure and employment vulnerability of coastal communities. If the Water Quality topic was expanded to Environmental Health, more proposals could be incorporated, including #9 , #14 , and #31 .

**Comment by demutsertlab** [comment]
I agree that #10 (my proposal) belongs to the fisheries focussed proposals.

**Comment by aesacco** [comment]
Perhaps we could also look into not only mapping fish distributions, but examine how the environment is changing through time. In this way, if we can automate these analytical processes, we can describe whether the Gulf is more-or-less static, or if it shows evidence of a transition due to climatic change. If we could get either 10 or 20 years of environmental data and study its change, then I think it would provide a more robust answer for determining what factors are responsible for distribution of our fish species. 

For data needs, then, I would suggest we look at _u_ and _v_ vector ocean currents, SST, chlorophyll a, freshwater flux into the ocean from the coast, salinity, and other data relevant to fish that we would be able to collect.

**Comment by kfrasier** [comment]
Has anyone worked with "Joint Species Distribution Models"? 
I've only very recently been made aware of them, but wondered if they might be  a way to unify our interests by incorporating multiple species into spatiotemporal models. 

Thorson, J.T.,et al . 2016. Joint dynamic species distribution models: a tool for community ordination and spatiotemporal monitoring. Glob. Ecol. Biogeogr. 25(9): 1144–1158. [(http://onlinelibrary.wiley.com/doi/10.1111/geb.12464/abstract.)]

I like the change over time idea @aesacco. I think we could get that environmental data very easily as far back as 2003 through Duke's MGET toolkit. Before then some of the satellite data gets sparse in my experience, but others may know of alternatives?

**Comment by aesacco** [comment]
@kfrasier the Joint SDM would be an interesting project. Perhaps it would be better suited for our situation, if we were to consider multiple fish species that have similar distributions or even behaviors , but even predator-prey interactions or species with similar resource needs would be beneficial to investigate. I think it all depends on how far (and how much) we can go/accomplish in the time we have. 

Duke does have some great datasets for ease of analyses, and there are also other level 3 products that we can get from NASA, NOAA, etc. I'll compile a few data portals that may be of interest over the weekend.

Thank you for the paper, @kfrasier.

**Comment by vdtobias** [comment]
I like the idea of looking at joint distributions over time and how changes in environmental factors affect them.  To play off @aesacco's idea of looking at fish with similar distributions, perhaps it would be good to add a layer to the project and determine which fish tend to be found together.  Maybe the joint SDM would already cover that, though.  I haven't looked into that method much yet.  Edit: Never mind- I can only read the abstract of the paper, but I see that they're using ordination to group the species so it would already make the groups.

**Comment by demutsertlab** [comment]
I haven't worked with Joint SDM but I do have developed a multi species food web model of the northern GOM coast of Louisiana win which I sued monthly layers of environmental parameters (model output)  to look at effects on fish and fisheries: 
De Mutsert, K., Steenbeek, J., Lewis, K., Buszowski, J., Cowan, J.H. Jr., and V. Christensen. 2016 Exploring effects of hypoxia on fish and fisheries in the northern Gulf of Mexico using a dynamic spatially explicit ecosystem model. Ecological Modelling 331: 142-150. doi:10.1016/j.ecolmodel.2015.10.013. Journal impact factor: 2.321

I think developing fish distribution and environmental parameter maps from data could be very useful for spatial validation for models like these.

**Comment by vdtobias** [comment]
It would be interesting to include how how the relationships change over time in addition to how fish are related to environmental factors.  That's one of the things we've been looking at for San Francisco Bay (where I'm working now).  For example, we have a couple of fish species whose abundance used to be strongly related to river outflow and the abundance of certain zooplankton, but in recent years those aren't the the dominant factors any more.

**Comment by ailich** [comment]
The joint species distribution model sounds very interesting as well as looking at how that compares with changes in the environment over time. One other potentially important factor to consider might be how fishing pressure changed over time as that might cause fish to become depleted in certain areas even if the environment was favorable.

**Comment by aesacco** [comment]
@ailich Fishing pressure; fishing effort could also suggest positive or negative effects on the population, especially if we look at these across pre- and post-DWH periods. What we should consider to either (a) find or (b) make ourselves, is a fish distribution/abundance layer from 90's or early 2000's environmental factors and older fisheries data (if it exists), and then we can have our baseline to use in recent past, present, and future projections of fish distribution. Something to think about. 

We should create a list of potential deliverables that we intend to produce for this synthesis and then we can start subtracting/adding as we hone the project through time. I've begun a section in the  summary post up top.

**Comment by demutsertlab** [comment]
This AFS news article, plus the reports listed in the references, seems very relevant to this project: https://fisheries.org/2017/06/habitat-science-is-an-essential-element-of-ecosystem-based-fisheries-management/

**Comment by aesacco** [comment]
@demutsertlab Yes, the article supports our current strategy for this project. I think if we characterize multispecies habitat interaction, we can add to a growing initiative thst can improve management of fish stocks. 
In addition, if we characterize the GoM through oceanography, environment, and atmospheric forcing change through time, these results could be used to investigate other species, fish or otherwise. A multiscale approach would also be very interesting to include if we want to understand the dynamic fisheries of the GoM, which will be important for any industrial development or resource mining in the area. I think we can expect many further applications of our results.

**Comment by adellapa** [comment]
I like the approach you guys are thinking about, as the potential products could be really useful to fishery and restoration managers. Since it looks like we would be interested in various cumulative impacts to fish, I thought the approach used in these papers might be useful and potentially applicable to our objectives: 
Halpern BS, and Fujita R. 2013. Assumptions, challenges, and future directions in cumulative impact analysis. Ecosphere 4(10): 131.
Halpern BS, Frazier M, Potapenko J, et al. 2015. Spatial and temporal changes in cumulative human impacts on the world’s ocean. Nature Communications 6: 7615 doi: 10.1038/ncomms8615
Just a thought

**Comment by tttang0602** [comment]
I like this project, it aims at the establishing a fish abundance map and connecting the too/bottom factors affecting the fish abundance. For example, the top factors could be predators such as dolphin and human intervention, to bottom factors could include the water quality and algaes.  Another quick thought is to also design analysis tools to measure the impact from the factors.  #20 (My proposal) studies more of the interaction between the fish population and their top factors. I would like to work on this project to improve my model later. In this project, I think we might only have time to explore a few factors. One question we need to address soon is which ones. (this could come from the most important thing this project wants to deliver.)

**Comment by vdtobias** [comment]
@tttang0602 has a good point about looking at how different factors affect distributions from both top-down and bottom-up.  I usually think of habitat factors (bottom-up influences) determining distributions, but the distribution of fishing pressure and predators (top-down) are super important for explaining why fish densities might be lower than expected in certain areas based on habitat factors alone.

**Comment by Yassin22** [comment]
Thank you everyone for sharing your thoughtful comments – great discussion. Based on the ongoing discussion and review of the proposals, it seems obvious that there is a strong link between water quality and fishery. Poor water quality would very likely impact the diversity and quantity of fish. I think establishing this link would greatly enhance the outcome of the project. We can explore fish spatial and temporal distribution in relation to water quality.
I envision that some of the data needed here would be water quality data such as physical, chemical, and biological data; and fish species spatial and temporal. My project Algae Blooms in the Gulf of Mexico: monitoring, prediction, impact, and mitigation (project #25) can play a significant role in this project. There is direct relationship between algae bloom and hypoxia (project #10). If you wish, we can include harmful algae blooms (HABs) and associated toxins as a subcomponent of water quality. HABs can be encountered in the GOM very frequently. As has been mentioned, we need to establish goals, tasks and approach to achieve our goals.
Here are some goals we may want to include:
-	Analysis and Synthesis of WQ impact on fish distribution
-	Modeling of fish distribution based on Environmental factors
-	Etc…
Tasks and approach:
-	Gather and analyze WQ and fishery data
-	Identify processes that influence fishery
-	Evaluate potential links between WQ and fishery
-	Prediction and validation of models
Here are my suggestions for the title: Water Quality Impact on Fish Distribution in the GOM
Or Synthesis of Water Quality and Fish Distribution in the GOM

**Comment by adellapa** [comment]
I think that the original objective to map fish distributions and environmental factors (more useful if conducted in a cumulative framework) is the more feasible. By identifying which of these factors may affect species distribution, reproductive success and habitat use, I can see how this information will be valuable to fisheries and restoration managers. For instance, this information can help better understand the potential impact of climate-related changes in environmental variables for fish survival and distribution. In turn, these maps could be overlayed with those of species-specific fish, stock, population distribution and essential fish habitats to identify the areas more in need of interventions and for which specific environmental factor. It would be great to also include an additional layer for fisheries impacts, but I feel this will require us to make further assumptions based on the species. Depending on the data source (fishery-dependent or fishery-independent surveys) and the species or stock, the use of CPUE may not be a valid proxy to consider for species abundance and distribution. Although would certainly add to the importance of the study, perhaps for the purpose of this project would be more effective and feasible to focus just on modeling and mapping species distribution based on forecasted changes in various environmental factors, and not consider fishery as an additional factor. Then, in the future, a second project could focus on addressing the impact of fishing effort and distribution, which will also be affected by climatic changes, on fish distribution.

**Comment by vdtobias** [comment]
I agree that the main focus of this project should be on the distribution of fish and/or fish communities and the environmental factors that drive the distribution.  I think that top-down factors such as fishing pressure can be considered an environmental factor from the perspective of the fish, though- it's just one more factor that could lend explanatory power to models linking fish and the environment.  I'm not very familiar with the survey data that we would be using for fish distributions, but if it's zero-rich we might need the extra information to explain why there are so many zeros.  Just something to keep in mind.

**Comment by adellapa** [comment]
Some additional food for thoughts for the potential analysis of these data:
Drymon JM, Carassou L, Powers SP, Grace MA, Dindo J, Dzwonkowski B. 2013. Multi-scale analysis of factors affecting the distribution of sharks throughout the Gulf of Mexico. Fishery Bulletin 111: 370-380.
The authors used PCA and co-inertia analysis (COIA) to look at the degree of agreement between CPUE of various shark species from fishery-independent surveys and environmental data taken from multiple database, including temperature, depth, Chl-a, salinity, dissolved oxygen, fish biomass, and crustacean biomass.

**Comment by paulacetus** [comment]
A study that may also give us some ideas for data sources & hypotheses (either complementary or follow-up) is the Pinsky et al 2013. Marine Taxa Track Local Climate Velocities. Science 341: 1239-1242. Authors used a 40-yr bottom survey dataset, which includes the Gulf of Mx. This study examines taxa range shifts as a function of environmental factors (primarily climate) and some biological characteristics.
An alternative to looking at distributional shifts, would be to examine trends in population abundance as a function of climate, oilspill, fishing pressure, etc. and instead of taking an assemblage approach, select representative taxa of different trophic levels and habitats. I think it may be less time consuming than preparing data for many species, especially if we select species for which there are stock assessments. For example, candidate species/groups could be: shrimp, forage fish (menhaden), reef fish (snappers and groupers), bottom fish (flounder, etc), pelagic fish (tunas, billfishes, etc), sharks and invasive species (lionfish).
This analysis could provide insights to managers on the sensitivity of these groups to concurrent environmental drivers that are significant in the Gulf of Mexico. Might be possible to implement as a forecasting tool in future endeavors.

**Comment by paulacetus** [comment]
A study that may also give us some ideas for data sources & hypotheses (either complementary or follow-up) is the Pinsky et al 2013. Marine Taxa Track Local Climate Velocities. Science 341: 1239-1242. Authors used a 40-yr bottom survey dataset, which includes the Gulf of Mx. This study examines taxa range shifts as a function of environmental factors (primarily climate) and some biological characteristics.
An alternative to looking at distributional shifts, would be to examine trends in population abundance as a function of climate, oilspill, fishing pressure, etc. and instead of taking an assemblage approach, select representative taxa of different trophic levels and habitats. I think it may be less time consuming than preparing data for many species, especially if we select species for which there are stock assessments. For example, candidate species/groups could be: shrimp, forage fish (menhaden), reef fish (snappers and groupers), bottom fish (flounder, etc), pelagic fish (tunas, billfishes, etc), sharks and invasive species (lionfish).
This analysis could provide insights to managers on the sensitivity of these groups to concurrent environmental drivers that are significant in the Gulf of Mexico. Might be possible to implement as a forecasting tool in future endeavors.

**Comment by ailich** [comment]
BOEM just recently released a high resolution bathymetry data set of the Northern Gulf. This may be a useful data set for predicting fish distributions especially more benthic fish

https://www.boem.gov/Gulf-of-Mexico-Deepwater-Bathymetry/
